src/test/java/racingcar/domain/DistanceTest.java

```diff
@@ -33,4 +34,13 @@ void testCarDistance() {
 
         assertThat(two.compareTo(twoTheSame)).isEqualTo(EQUAL);
     }
+
+    @Test
+    void testIncrement() {
+        Distance one = new Distance(DISTANCE_ONE);
+        Distance two = new Distance(DISTANCE_TWO);
+        one.increment();
+
+        assertThat(one.compareTo(two)).isEqualTo(EQUAL);
```

Comment:
```suggestion
        Distance one = new Distance(DISTANCE_ONE);
        one.increment();

        assertThat(one).isEqualTo(new Distance(DISTANCE_ONE + 1));
```
이런 방법으로 테스트의 내용을 바꿔보면 어떨까요?